lib/generators/komponent/install_generator.rb

```diff
@@ -21,7 +23,15 @@ def move_webpacker_default_structure
 
       def create_komponent_default_structure
         empty_directory(components_directory)
-        create_file(components_directory.join("index.js"))
+        template = <<-eos
+import { Application } from "stimulus";
+import { autoload } from "stimulus/webpack-helpers";
+
+const application = Application.start();
+const context = require.context("./", true, /_controller\.js$/);
+autoload(context, application);
+        eos
```

Comment:
By doing that we make it harder to generate different javascript packs (for example we may want a pack with admin components, not included in all layouts). If someone wants different packs he would have to rewrite this part.

I think we should instead move that into the component. The generated stimulus component would add this code in its own js and would autoload all the controllers existing in its own directory.

I'm not sure it would work if we create multiple `Application` though. So maybe we need a `stimulus_application` module or something that would generate the common `application`.

Reply:
this is already in component/index.js

Reply:
In component specific file, like I say in my comment. I copy-paste here.

When we run install generator, create a `frontend/stimulus_application.js` file:

```js
import { Application } from "stimulus";
const application = Application.start();
export {application as default};
```

When we generate a new component, keep the `[component_name].js`, and append to that file:

```js
import application from 'stimulus_application';
const context = require.context('./', true, /_controller\.js$/);
autoload(context, application);
```

Reply:
ok, I will apply it

Reply:
@florentferry done.